src/features/MentorPage/MentorChips/ShowMoreChips.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components';
+import ExpandIcon from '../../../static/img/icon-expand.svg';
+
+/**
+ * Show more button style and layout
+ * Should update Skill Chips container when pressed?
+ */
+
+const ShowMoreButtonHandler = (event: React.MouseEvent<HTMLDivElement>) => {
+  const showMoreButton = document.getElementById('skill-chips');
+  if (showMoreButton) {
+    showMoreButton.classList.toggle('show-more');
+    console.log(event.currentTarget);
+    event.currentTarget.innerText === 'Näytä kaikki aiheet'
```

Comment:
Using the boolean state-variable we can also get rid of this, and use the value instead.